crates/proto/src/serialize/txt/root.key

```diff
@@ -0,0 +1,4 @@
+; $ dig DNSKEY .
```

Comment:
I don't think we have other test data in the `src` tree? Prefer putting this somewhere in the `tests` hierarchy.